04/app.js

```diff
@@ -0,0 +1,38 @@
+
+function createArray (min, max) {
+
+    const arr = []
+    for(let i = 0; i < 20; i++) {
+        arr.push(Math.round((Math.random() * (max - min)) + min))
+    }
+
+    return arr;
+}
+
+function getLargest (arr) {
+    
+    sortArr = arr.sort(function(a,b){
+        return b - a;
+    });
+
+    return sortArr.slice(0, 10);
+}
+
+function getAvg (arr) {
+
+    let sum = 0;
+    arr.forEach(function(element) {
+        sum += element / 10;
+    });
+
+    return sum;
```

Comment:
Lepiej jest dodać, a potem dzielić.
W programowaniu ma to na tyle znaczenie, że liczby są zapisywane w systemie dwójkowym i może to powodować pewne rozbieżności w liczeniu. Zobacz: `0.1 + 0.2` (nie mówiąc już o dzieleniu)